dev/com.ibm.ws.jdbc_fat_postgresql/test-applications/postgresqlApp/src/jdbc/fat/postgresql/web/PostgreSQLTestServlet.java

```diff
@@ -217,6 +224,60 @@ public void testDefaultAutoCommit() throws Exception {
         }
     }
 
-    // TODO: Add a test for defaultAutoCommit setting in a global transaction
+    @Test
```

Comment:
This test case lacks a description. Please add one now because it makes more sense to do so when the test was just written than to have someone go back and figure it out again when it is time to submit the Feature Test Summary.